stable/prometheus-adapter/README.md

```diff
@@ -104,37 +104,39 @@ The command removes all the Kubernetes components associated with the chart and
+| `rbac.create`           | If true, create & use RBAC resources                                                  | `true`                                      |
+| `resources`             | CPU/Memory resource requests/limits                                                   | `{}`                                        |
+| `rules.default`         | If `true`, enable a set of default rules in the configmap                             | `true`                                      |
+| `rules.custom`          | A list of custom configmap rules                                                      | `[]`                                        |
+| `rules.existing`        | The name of an existing configMap with rules. Overrides default, custom and external. | ``                                          |
+| `rules.external`        | A list of custom rules for external metrics API                                       | `[]`                                        |
+| `rules.resource`        | `resourceRules` to set in configmap rules                                             | `{}`                                        |
+| `service.annotations`   | Annotations to add to the service                                                     | `{}`                                        |
+| `service.port`          | Service port to expose                                                                | `443`                                       |
+| `service.type`          | Type of service to create                                                             | `ClusterIP`                                 |
+| `serviceAccount.create` | If true, create & use Serviceaccount                                                  | `true`                                      |
+| `serviceAccount.name`   | If not set and create is true, a name is generated using the fullname template        | ``                                          |
+| `tls.enable`            | If true, use the provided certificates. If false, generate self-signed certs          | `false`                                     |
+| `tls.ca`                | Public CA file that signed the APIService (ignored if tls.enable=false)               | ``                                          |
+| `tls.key`               | Private key of the APIService (ignored if tls.enable=false)                           | ``                                          |
+| `tls.certificate`       | Public key of the APIService (ignored if tls.enable=false)                            | ``                                          |
+| `extraVolumeMounts`     | Any extra volumes mounts                                                              | `[]`                                        |
+| `extraVolumes`          | Any extra volumes                                                                     | `[]`                                        |
```

Comment:
Add a section to README about how to use with secured Prometheus

Reply:
I added a sentence about Prometheus under HTTPS and played a bit with the formatting for more clarity. 

I was a bit reluctant to add examples as those are already in `values.yaml`. What do you think? Is it enough like this?